lichess-bot.py

```diff
@@ -685,24 +687,35 @@ def print_pgn_game_record(li: lichess.Lichess, config: Configuration, game: mode
     game_path = os.path.join(config.pgn_directory, game_file_name)
 
     lichess_game_record = chess.pgn.read_game(io.StringIO(li.get_game_pgn(game.id)))
+    if not isinstance(lichess_game_record, chess.pgn.Game):
+        return
     try:
         # Recall previously written PGN file to retain engine evaluations.
         with open(game_path) as game_data:
             game_record = chess.pgn.read_game(game_data)
```

Comment:
It would be better to set `game_record` to `lichess_game_record` rather than returning early:

`game_record = chess.pgn.read_game(game_data) or lichess_game_record`